src/pathfora.js

```diff
@@ -694,6 +705,40 @@
     },
 
     /**
+     * @description check if a customTrigger widget
+     * is ready to be displayed, and if so display the widget
+     * @param {object} widget
+     */
+    triggerWidget: function (widget) {
```

Comment:
naming is probably fine here but if you are looking to change it could use something like `execute` or `evaluate` ... even though its a big wordy at least triggerWidget is accurate and hard to confuse with any of the other functions we have.